### ISSUE TITLE
Request: Add settings option to supress Any types in favour of Strings.

### ISSUE THREAD

**Issue by occultus73** [wuseal/JsonToKotlinClass#300]
The reason being I can't  parcelize a data class with Any types in them.

Cheers.

**Comment by wuseal** [comment]
@occultus73 Hi, Can you show some examples or demos?

**Comment by occultus73** [comment]
Oh it's very simple really: when the plugin is fed null json property, it cannot of course infer the data type from a null so the corresponding Kotlin property that is generated has a type of "Any"

Changing nulls to produce String instead of Any for null would only be a matter of one carefully placed IF statement somewhere, I would have thought... and a new button in settings.

**Comment by wuseal** [comment]
@occultus73 Got it，But maybe null is a real object in future API response, Then May JSONObject can't cast to String, And It will broken in program, all right?

**Comment by wuseal** [comment]
@occultus73 It's easy to achieve your goal, Just create an [Extension](https://github.com/wuseal/JsonToKotlinClass/blob/master/doc_for_extensions.md) for the plugin is OK.
Change Type `Any` to `String` when [Property#originJSONValue](https://github.com/wuseal/JsonToKotlinClass/blob/8edda826d818b34d28dc77c806b5b21e297daed7/src/main/kotlin/wu/seal/jsontokotlin/model/classscodestruct/Property.kt#L7) is `null`

**Comment by occultus73** [comment]
That is true. I think 9/10 instances it's a string. But then again over a very large data model that 1/10 is bound to come along. 

Perhaps this setting wouldn't be so helpful after all. 😐

**Comment by wuseal** [comment]
@occultus73 Haha, Welcome to create a pull request for this, It's also an acceptable option to replace Any with String when json value is nul
